code/src/WijmoProvider/Columns/AbstractProviderColumn.ts

```diff
@@ -121,9 +122,11 @@ namespace WijmoProvider.Column {
                         ? providerGrid.columns.length
                         : indexPosition;
 
-                this.provider = new wijmo.grid.Column(this.getProviderConfig());
-
-                providerGrid.columns.insert(indexPosition, this.provider);
+                this.provider = new wijmo.grid.ColumnGroup(
+                    this.getProviderConfig()
+                );
+                //@ts-ignore
```

Comment:
why do we need this ts-ignore?

Reply:
Basically, because wijmo defined column groups as columnGroups: any[], even though they extend ObservableArray. So, since its any, it does not recognize the insert method.

Reply:
Using a cast here is not an option, right?
And why are we using the providerGrid instead of this.provider?